### ISSUE TITLE
Unable to generate Gradle project with Gatling enabled 

### ISSUE THREAD

**Issue by fezu54** [jhipster/jhipster-kotlin#46]
When I try to generate a Microservice gateway project with Gradle as build tool and Gatling support enabled, the generation will fail. Generating the same project with Maven as build tools works.

Reproduce the error
```bash
Which *type* of application would you like to create? Microservice gateway
? What is the base name of your application? gateway
Trying to use blueprint generator-jhipster-kotlin
? As you are running in a microservice architecture, on which port would like your server to run? It should be unique to avoid port confl
icts. 8080
? What is your default Java package name? com.test.kotlin
? Which service discovery server do you want to use? JHipster Registry (uses Eureka, provides Spring Cloud Config support and monitoring 
dashboards)
? Which *type* of authentication would you like to use? JWT authentication (stateless, with a token)
? Which *type* of database would you like to use? SQL (H2, MySQL, MariaDB, PostgreSQL, Oracle, MSSQL)
? Which *production* database would you like to use? MariaDB
? Which *development* database would you like to use? MariaDB
? Do you want to use Hibernate 2nd level cache? Yes
? Would you like to use Maven or Gradle for building the backend? Gradle
? Which other technologies would you like to use? Search engine using Elasticsearch
Trying to use blueprint generator-jhipster-kotlin
WARNING! No blueprint found for client falling back to default generator
? Which *Framework* would you like to use for the client? Angular 6
? Would you like to enable *SASS* stylesheet preprocessor? Yes
? Would you like to enable internationalization support? Yes
? Please choose the native language of the application English
? Please choose additional languages to install Dutch, French, German, Italian, Japanese, Polish, Spanish
? Besides JUnit and Jest, which testing frameworks would you like to use? Gatling, Cucumber, Protractor
? Would you like to install other generators from the JHipster Marketplace? No
```

```
events.js:183
      throw er; // Unhandled 'error' event
      ^

Error: ERROR! Copying template ../../../node_modules/generator-jhipster/generators/server/templates/gradle/gatling.gradle.ejs failed. [Error: ENOENT: no such file or directory, open '/usr/lib/node_modules/generator-jhipster-kotlin/node_modules/generator-jhipster/generators/server/templates/gradle/gatling.gradle.ejs']
    at Environment.error (/usr/lib/node_modules/generator-jhipster/node_modules/yeoman-environment/lib/environment.js:157:40)
    at module.exports.error (/usr/lib/node_modules/generator-jhipster-kotlin/node_modules/generator-jhipster/generators/generator-base.js:1978:18)
    at ejs.renderFile (/usr/lib/node_modules/generator-jhipster-kotlin/node_modules/generator-jhipster/generators/utils.js:197:23)
    at tryHandleCache (/usr/lib/node_modules/generator-jhipster-kotlin/node_modules/generator-jhipster/node_modules/ejs/lib/ejs.js:254:14)
    at Object.exports.renderFile (/usr/lib/node_modules/generator-jhipster-kotlin/node_modules/generator-jhipster/node_modules/ejs/lib/ejs.js:482:10)
    at Object.renderContent (/usr/lib/node_modules/generator-jhipster-kotlin/node_modules/generator-jhipster/generators/utils.js:193:9)
    at module.exports.template (/usr/lib/node_modules/generator-jhipster-kotlin/node_modules/generator-jhipster/generators/generator-base-private.js:636:23)
    at module.exports.writeServerBuildFiles (/usr/lib/node_modules/generator-jhipster-kotlin/generators/server/files.js:197:26)
    at Object.<anonymous> (/usr/lib/node_modules/generator-jhipster-kotlin/node_modules/generator-jhipster/node_modules/yeoman-generator/lib/index.js:399:25)
    at /usr/lib/node_modules/generator-jhipster-kotlin/node_modules/run-async/index.js:25:25
```

Related issues
Link to PR (if any)
JHipster Version(s) --> 5.3.4
[x] Checking this box is mandatory (this is just to show you read everything)

**Comment by sendilkumarn** [comment]
@fezu54 Oops sorry, I think that file is removed from the master. Are you using `master` of JHipster. I would suggest use v5.3.0 instead of that particular version

**Comment by sendilkumarn** [comment]
Closing this since there is no activity, hope the issue is fixed with v5.3.0 

Feel free to open this ticket if you still have the issue.
